### DISCUSSION TITLE
Adding text / image overlays through Flutter

### DISCUSSION THREAD

**Discussion by dylandamsma** [shogo4405/HaishinKit.dart#61] [Q&A]
Hi there!

First of all, great work @shogo4405 

I would love to be able to add Text / Image overlays. My main use-case is streaming tennis matches, where I want to show the live score of the match in the video feed. I see that the Haishinkit for iOS & Android can do this (is that correct?), so at the moment I am thinking of forking the library and trying to implement it myself. 

Before I do so - can I have a confirmation that the Offscreen Rendering affects the actual stream output to Youtube/Facebook?

Thanks 🙏

**Comment by shogo4405** [comment]
Yes, both iOS and Android support such a feature.The Flutter updates are currently lagging behind.
Does this sound good? I have forgotten how to use Facebook, so it's difficult to experiment with it immediately.

<img width="1210" alt="スクリーンショット 2024-07-18 20 49 07" src="https://github.com/user-attachments/assets/5b72cb6b-b733-4ed7-b297-20e7021d56a1">
